### PR TITLE
Graphical Patch Stuff, and new strings. 1016 remaining in this fork.

### PR DESCRIPTION
**Pull request by Sterophonick**
This contains the most strings out of all of these forks, and there are some tools to localize the graphics, along with assembly code for that.

### CODE REVIEW

**Comment by Sterophonick** [comment]
Oh yeah, it also has a patch for incorporating Mato's Patch, and NTSC Colors for Mother 1. Update Readme as well.

**Comment by jeffman** [comment]
Thanks! A few questions:
- Is there anything in the Graphics Patch folder that's included in the build, or is it just reference material? I'd like to keep the repo clean if possible; there's a bunch of tool-related stuff that doesn't look necessary.
- In insert.bat, the script tool command looks off. It should come after the copy command and the second-last parameter should be the EB ROM instead of the fresh M12 ROM

**Comment by Sterophonick** [comment]
Jeffman

The Graphics Patch Folder is for the source on the graphical patches. The gfx folder contains the converted RAW data for the image changes.

What is the point of the Earthbound ROM?

**Comment by PhoenixBound** [comment]
@Sterophonick The EarthBound ROM is used to decompile an eb-strings.txt file I think. The non-GUI ScriptTool was used to dump all the "strings" files.

**Comment by PhoenixBound** [comment]
About the merge conflict: to fix it, you should be able to use [this command](https://stackoverflow.com/a/16054884) from StackOverflow:

`git merge origin/master --no-commit -s recursive -X renormalize`

and then fix the rest of the conflicts by editing the strings file manually and committing.

File: working/m12-strings-english.txt [hunk 1]
```diff
@@ -1281,11 +1532,11 @@
...
 ^L1288^@[C2 FF][1A FF 05 00] tried to get away,[1B FF 3C 00][02 FF]@and did![02 FF][00 FF]
 ^L1289^@[9F FF] relaxed[01 FF]  and took a deep breath.[02 FF][00 FF]
 ^L1290^@The Suporma sang the song[01 FF]  <Ode to Orange Kid.>[02 FF]@As soon as it finished, the machine[01 FF]  broke down.[6D FF FF 00 94 00][1D FF][00 FF]
-^L1291^@By using the Insignificant Present, you[01 FF]  had a very fruitful experience[02 FF]@that cannot be understood by someone[01 FF]  who does not use something[01 FF]  insignificant.[1D FF][00 FF]
+^L1291^@By using the Insignificant [01 FF]  Present, you had a very fruitful[01 FF]  experience, that cannot be[01 FF]  understood by someone who[01 FF]  does not use something[01 FF]  insignificant.[1D FF][00 FF]
```
Comment (PhoenixBound) [inline working/m12-strings-english.txt]: This line is missing an 02 FF after it got fixed.

File: working/m12-strings-english.txt [hunk 2]
```diff
@@ -2096,53 +2387,64 @@
...
-^L2123^[D4 FF 67 00][C1 FF][87 FF]@[AD FF] realized the power of[01 FF]  [2D FF].[22 FF][00 FF]
+^L2115^@[86 FF _L3626_]Defense went up by [AC FF]![22 FF][00 FF]
+^L2116^@[86 FF _L3626_]Speed went up by [AC FF]![22 FF][00 FF]
+^L2117^@[86 FF _L3626_]Guts went up by [AC FF]![22 FF][00 FF]
+^L2118^@[86 FF _L3626_]Vitality went up by [AC FF]![22 FF][00 FF]
+^L2119^@[86 FF _L3626_]IQ went up by [AC FF]![22 FF][00 FF]
+^L2120^@[86 FF _L3626_]Luck went up by [AC FF]![22 FF][00 FF]
+^L2121^@[86 FF _L3627_]Maximum HP went up by [AC FF]![22 FF][00 FF]
+^L2122^@[86 FF _L3628_]Maximum PP went up by [AC FF]![22 FF][00 FF]
+^L2123^[D4 FF 67 00][C1 FF][87 FF]@[AD FF] learned the power of[01 FF]  [2D FF].[22 FF][00 FF]
```
Comment (PhoenixBound) [inline working/m12-strings-english.txt]: The original game used the text "realized" here and in a few other places, so you'll either want to be consistent with them or change them to consistent with this.

File: working/m12-strings-english.txt (hunk 1 above)
Comment (Sterophonick) [reply working/m12-strings-english.txt]: ok.

File: CONTRIBUTORS.txt [hunk 3]
```diff
@@ -14,6 +14,16 @@ Text conversion:
...
   PhoenixBound
   Pidgin
+  Sterophonick
+  Karmageddon
 
 Special thanks:
-  TragicManner
\ No newline at end of file
+  TragicManner
+  Nintendo
```
Comment (jeffman) [inline CONTRIBUTORS.txt]: I'd lean towards not including the companies in the contributors

Comment (Karmageddon00) [reply CONTRIBUTORS.txt]: Ditto.

File: working/m12-strings-english.txt [hunk 4]
```diff
@@ -8,21 +10,34 @@
...
+^L2222^@Okay, okay... I won't say[01 FF]  anything to hurt you.[02 FF]@Will you please[01 FF]  come with me?[01 FF][22 FF][03 FF 00 00][04 FF _L2220_][05 FF _L2221_][80 FF _L2221_]
+^L2220^[08 FF 60 00]@Ok,[1B FF 0F 00] good buddy![1B FF 14 00] Let's blow[01 FF]  this popsicle stand...[02 FF]@Before we go,[1B FF 0F 00] why don't you[01 FF]  say <goodbye> to your mom.[DB FF 11 00 02 00 0E 00][CE FF 11 00 00 00][1B FF 0A 00][01 FF]  Don't you agree,[1B FF 0F 00] ma'am?[A1 FF 0E 00 0D 00][1D FF][00 FF]
+^L17^[1C FF 49 00 _L2229_][1C FF 73 02 _L2230_][1C FF 4C 00 _L2231_]@I'll do everything I can to help[01 FF]  you![1B FF 14 00]  Good luck on your[01 FF]  adventure.[02 FF]@You might get hungry along the[01 FF]  way,[1B FF 0F 00] so here is a Cookie.[22 FF][91 FF FF][82 FF _L2232_][93 FF 00 54][08 FF 4C 00][86 FF _L2233_][02 FF]@You can also leave anything[01 FF]  you don't need with me.[02 FF]@Do you want me to hang on to[01 FF]  something?[01 FF][03 FF 00 00][04 FF _L2234_][05 FF _L2235_][00 FF]
+^L2235^@Be careful![1D FF][00 FF]
+^L2234^[86 FF _L2236_][00 FF]
+^L18^[1C FF FB 02 _L2214_][1C FF 49 00 _L2215_][00 FF]
```
Comment (PhoenixBound) [inline working/m12-strings-english.txt]: This 00 FF is what made the Dad call at the beginning of the game not happen, if you wanted to try to continue further.

Comment (Sterophonick) [reply working/m12-strings-english.txt]: Fixed.

File: CONTRIBUTORS.txt (hunk 3 above)
Comment (Sterophonick) [reply CONTRIBUTORS.txt]: Understandable.

**Comment by Sterophonick** [comment]
Holy crap the amount of labels in this fork went from 1016 to 266.

**Comment by PhoenixBound** [comment]
It, uh, looks like almost all of the main label changes from my PR got accidentally undone in the confusion :/

There's also a bunch of 00 FF's in places where they shouldn't be, at least towards the beginning.

(I'm not a fan of the one-label-per-line thing 'cause I almost exclusively use the tool and it doesn't match the M12 text anymore, but I know that was a conscious decision, so I won't focus on it that much for now.)

**Comment by jeffman** [comment]
I think this has diverged too much at this point. We can manually go through and pick out the pieces we want to merge and perhaps create a new PR for those. Closing for now.

**Pull request closed without merging**